projects/buttons/fastener/main.py

```diff
@@ -12,7 +12,7 @@
 import re
 
 # generators
-def id_generator(size=4, chars=string.ascii_lowercase + string.digits):return ''.join(random.choice(chars) for _ in range(size))
+def id_generator(size=4, chars=string.ascii_lowercase + "labs"):return ''.join(random.choice(chars) for _ in range(size))
```

Comment:
can we up the size here to 10

Reply:
this affects an instancehandler.py conditional, see comment there